src/client/actions/UserNoteUpdate.js

```diff
@@ -21,6 +21,7 @@ class UserNoteUpdateAction extends Action {
 
 /**
  * Emitted whenever a note is updated.
+ * <warn>This is only available when using a user account.</warn>
```

Comment:
Can you remove that?

We are removing userbot support in https://github.com/discordjs/discord.js/pull/2559 so this is redundant and will only create merge conflicts in that other PR.